inc/policybase.class.php

```diff
@@ -59,6 +59,34 @@
    protected $policyData;
 
    /**
+    * @var array list of statuses specific to a policy type. To be overrided in child class
+    */
+   protected $specificStatuses = [];
+
+    /**
+     * get common task statuses
+     * @return array
+     */
+   public static final function getEnumBaseTaskStatus() {
+      return [
+         'pending'   => __('Pending', 'flyvemdm'),
+         'received'  => __('Received', 'flyvemdm'),
+         'done'      => __('Done', 'flyvemdm'),
+         'failed'    => __('Failed', 'flyvemdm'),
+         'canceled'  => __('Canceled', 'flyvemdm'),
+      ];
```

Comment:
@rafaelje: here is the list of status common to all policies
- pending => not sent yet to the agent
- received => the agent acknoledge recepit of the task
- done => the agent completed the task (successfully)
- failed => the agent tried and failed to complet ethe task
- canceled => the backend canceled the task (and set this status itself) The agent should not set we don't have use case for that, but why not)

Reply:
I'll implement this in the Android Agent, now the agent has generic response